memcached.c

```diff
@@ -13785,6 +13873,11 @@ bool conn_nread(conn *c)
         return true;
     }
 
+    if (c->rlbytes == 0) {
+        c->ritem   = c->rlist[c->rlcurr].iov_base;
+        c->rlbytes = c->rlist[c->rlcurr].iov_len;
```

Comment:
```suggestion
        c->ritem = c->rlist[c->rlcurr].iov_base;
        c->rlbytes = c->rlist[c->rlcurr].iov_len;
```
정렬하지 않는 편이 나은 것 같습니다.